syntax = "proto3";
package m2g.pb;

enum ResourceType {
  NO_RESOURCE = 0;

  // For HUMAN_PLAYERs
  MONEY = 1;
  ERA = 2;

  // For TILEs and MERCHANTs
  COAL_CUBE_COUNT = 3;
  IRON_CUBE_COUNT = 4;
  BEER_BARREL_COUNT = 5;
}
